src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -404,9 +404,23 @@ public void TestSignaturesV2_3_X()
             Assert.IsType<RelationalGroupedDataset>(_df.GroupBy(_df["age"]));
             Assert.IsType<RelationalGroupedDataset>(_df.GroupBy(_df["age"], _df["name"]));
 
-            Assert.IsType<DataFrame>(_df.GroupBy("name").Sum("age"));
-            Assert.IsType<DataFrame>(
-                _df.WithColumn("tempAge", _df["age"]).GroupBy("name").Sum("age", "tempAge"));
+            var relationalGroupedDataset = _df.GroupBy("name");

```

Comment:
Let's enclose this whole subsection with a { }.  We can shorten the var names.  Just using `df1` and `df2` should suffice.

Reply:
Gotcha! Done